sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1943,7 +1955,34 @@ cl_int ExecCGCommand::enqueueImp() {
       }
     }
 
-    MQueue->getThreadPool().submit<DispatchHostTask>(DispatchHostTask(this));
+    std::vector<interop_handle::ReqToMem> ReqToMem;
+    // Extract the Mem Objects for all Requirements, to ensure they are
+    // available if a user ask for them inside the interop task scope
+    const std::vector<Requirement *> &HandlerReq = HostTask->MRequirements;
+    auto ReqToMemConv = [&ReqToMem, HostTask, this](Requirement *Req) {
```

Comment:
Fix me if I am wrong, but having 'this' in this list seems redundant.

Reply:
Looks like that. Removed.